From 159bfa4ec094e7d342c9b59c31bfea7dccbac58a Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Sat, 7 Feb 2015 17:24:48 -0800
Subject: [PATCH 01/10] doc: add a couple olinks to fsproto & xfs-design docs

Don't seem to have ability to link to BDF or Xserver internals docs yet

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 doc/fontlib.xml | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/doc/fontlib.xml b/doc/fontlib.xml
index c5079c8..ce31bb5 100644
--- a/doc/fontlib.xml
+++ b/doc/fontlib.xml
@@ -111,8 +111,9 @@
       </listitem>
       <listitem>
 	<para>
+	  <olink targetdoc='xfs-design' targetptr='xfs-design'>
 	  <citetitle pubwork="article">Font Server Implementation
-	    Overview</citetitle> which discusses the design of the font
+	    Overview</citetitle></olink> which discusses the design of the font
 	  server.
 	</para>
       </listitem>
@@ -126,7 +127,9 @@
       </listitem>
       <listitem>
 	<para>
+	  <olink targetdoc='fsproto' targetptr='fsproto'>
 	  <citetitle pubwork="article">The X Font Service Protocol</citetitle>
+	  </olink>
 	  for a description of the constraints placed  on the design by
 	  including support for this font service mechanism.
 	</para>
-- 
2.8.3

